datafusion/core/Cargo.toml

```diff
@@ -27,7 +27,7 @@ homepage = { workspace = true }
 repository = { workspace = true }
 license = { workspace = true }
 authors = { workspace = true }
-rust-version = "1.70"
+rust-version = "1.71"
```

Comment:
Could we take this opportunity to set this as same as workspace? e.g.

https://github.com/apache/arrow-datafusion/blob/7a5f2054305fd92852b589473afbc9bb034379d7/datafusion/physical-plan/Cargo.toml#L29

Unless its preferable to keep these separate? (even though they currently are the same)

Reply:
That is an excellent idea.  I vaguely (but perhaps incorrectly) remember something related to releasing was tricky with MSRV -- @andygrove  do you remember? Would it be ok to switch the sub crates to use the workspace version?

Reply:
Done in 9280b7a166814bc6f3660c7471fb9858ac8bde3a

Reply:
Ok, I found out again. The reason the version is replicated is that `cargo msrv verify` doesn't support workspace versions

Thus if you do `cd datafusion/core && cargo msrv verify`, it fails like 

```
Fetching index
Unable to find key 'package.rust-version' (or 'package.metadata.msrv') in '/Users/andrewlamb/Software/arrow-datafusion/Cargo.toml'
```

I have fixed that

Reply:
Tracking issue for that in cargo msrv: https://github.com/foresterre/cargo-msrv/issues/590

Reply:
Thanks @Jefffrey  -- I have added an issue ref as a comment so we don't forget it in the future